### DISCUSSION TITLE
Help with basic captioning and filewords usage🙏🤧😟

### DISCUSSION THREAD

**Discussion by wktra** [d8ahazard/sd_dreambooth_extension#382] [Q&A]
I'm trying to train a cat-girl character of my own creation. She has cat ears, a black tail, whiskers and three ponytails on her head.

Can anyone please answer the following questions to help me move forward? 🥺

1) what is an example of how to name each training image? Can someone please give me a concrete example of how to structure each file with instance,  class, and description? It's the actual punctuation and format I have trouble with. 

2) based on the above answer, can anyone please help me fill out the file words section properly? I'm totally confused as to how to do this. Is "filewords" an actual text file I have to fill out in the training template directory? 

So far, I've just been filling out the instance prompt and class section with "ctgurlx, girl, solo" and "1girl, solo" for the class. I've also provided my own class images. 

But despite the character design style looking great, sometimes the generations miss either the cat tail or the ears or the 3 ponytails.

Can anyone please help me come up with a good routine for this character using filewords and captions for this particular cat-girl character? 🙏🥺

Thanks in advance.

**Comment by piyarsquare** [comment]
Thanks for sksing this question.
I too have been using only the instance and class prompts. 
I am not sure what to do with filewords and hope to make better use of this excellent tool.
I hope you draw an answer.
